src/test/java/nl/tudelft/ti2206/group9/util/LoggerTest.java

```diff
@@ -32,23 +32,27 @@ public void setUp() {
     }
 
     @Test
-    public void testWriteToFile() throws IOException {
-        final GameUpdate testUpdate =
-                new GameUpdate(Category.PLAYER, Player.JUMP);
-        final String testUpdateString = "[PLAYER] Jumping.\n";
-
-        logger.update(null, testUpdate);
-        logger.writeToFile();
-        String log = new String(Files.readAllBytes(Paths.get(TESTLOG)),
-                StandardCharsets.UTF_8).substring(Logger.FORMAT.length() - 1);
-        assertEquals(testUpdateString, log);
-
-        // append = false, thus a new file should be created
-        logger.update(null, testUpdate);
-        logger.writeToFile(false);
-        log = new String(Files.readAllBytes(Paths.get(TESTLOG)),
-                StandardCharsets.UTF_8).substring(Logger.FORMAT.length() - 1);
-        assertEquals(testUpdateString, log);
+    public void testWriteToFile() {
+        try {
+            final GameUpdate testUpdate =
+                    new GameUpdate(Category.PLAYER, Player.JUMP);
+            final String testUpdateString = "[PLAYER] Jumping.\n";
+
+            logger.update(null, testUpdate);
+            logger.writeToFile();
+            String log = new String(Files.readAllBytes(Paths.get(TESTLOG)),
+                    StandardCharsets.UTF_8).substring(Logger.FORMAT.length() - 1);
+            assertEquals(testUpdateString, log);
+
+            // append = false, thus a new file should be created
+            logger.update(null, testUpdate);
+            logger.writeToFile(false);
+            log = new String(Files.readAllBytes(Paths.get(TESTLOG)),
+                    StandardCharsets.UTF_8).substring(Logger.FORMAT.length() - 1);
+            assertEquals(testUpdateString, log);
+        } catch (IOException e) {
+            fail();
```

Comment:
When the test fails at this point, is it easy to see why? Because if it's not, you should add a message to the method like `fail("IOException in test <...>");

<review, logic